src/main/java/com/example/hyperlocal/Merchant.java

```diff
@@ -0,0 +1,47 @@
+package com.example.hyperlocal;
```

Comment:
Fix package name. It should not contain example.
com.hyperlocal.server seems better name

Reply:
Fixed it. Thanks a lot!!